715615 - ipmitool -o list return always 1

ipmitool -o list should return zero exit code

diff -up ipmitool-1.8.11/lib/ipmi_main.c.o-exitcode ipmitool-1.8.11/lib/ipmi_main.c
--- ipmitool-1.8.11/lib/ipmi_main.c.o-exitcode	2012-02-20 17:38:46.790553731 +0100
+++ ipmitool-1.8.11/lib/ipmi_main.c	2012-02-20 17:38:46.810553612 +0100
@@ -456,6 +456,7 @@ ipmi_main(int argc, char ** argv,
 			if (strncmp(oemtype, "list", 4) == 0 ||
 			    strncmp(oemtype, "help", 4) == 0) {
 				ipmi_oem_print();
+				rc = 0;
 				goto out_free;
 			}
 			break;
